invoice/src/main/java/org/killbill/billing/invoice/model/DefaultInvoice.java

```diff
@@ -158,6 +161,17 @@ public boolean addInvoiceItems(final Collection<InvoiceItem> items) {
         return invoiceItems;
     }
 
+    @Override
+    public List<String> getTrackingIds() {
+        return trackingIds;
+    }
+
+    @Override
+    public boolean addTrackingIds(final Collection<String> trackingIds) {
+        this.trackingIds.addAll(trackingIds);
```

Comment:
You could have returned the result of `addAll` to make the returned value a bit more meaningful. This (historic) boolean API is a bit weird though.

Reply:
Ack - will fix